test/system/screenshots_test.rb

```diff
@@ -61,6 +61,22 @@ class ScreenshotsTest < ApplicationSystemTestCase
     take_screenshot
   end
 
+  test 'login modal form validation' do
+    visit '/'
+    click_on 'Login'
+
+    fill_in 'user_session[username]', with: 'Bob'
+    fill_in 'user_session[password]', with: 'invalid'
+
+    click_on 'Log in'
+
+    # Get the error message (remove '×' that closes the modal and remaining whitespaces)
+    error_msg = find('.error-msg-container').text.gsub('×', '').strip()
+
+    assert_equal( error_msg, 'Invalid username or password' )
```

Comment:
Yeah this is good! Could you explain with a comment just above that the password is invalid because there are less than 8 characters? Thanks

Reply:
:+1: